routes/i.js

```diff
@@ -33,6 +33,8 @@ let isDeletePost = (req) => { return req.body.deleted % 2 == 1; };
 let getPageOwner = async (req) => { return await User.findOne({where: {username: req.params.username}}) };
 let getMe = async (req) => { return await User.findByPk(req.user.id) };
 
+const limitString = (str, length) => { return Array.from(str).slice(0, length).join(); };
```

Comment:
```suggestion
const limitString = (str, length) => { return Array.from(str).slice(0, length).join(''); };
```
少なくともここは直さないと動かない気がしました。

Reply:
joinで引数省略したときはコンマが入るの忘れてました 🙇 
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/join